japan/README.md

```diff
@@ -0,0 +1,18 @@
+# Kubernetes Contributor Playground for Japan
+
+Hello new contributors!
+
+新コントリビュータの皆さんこんにちわ！
+
+This repository will be used as a safe space for participants in the New Contributor Onboarding Track to familiarize themselves with (some of) the Kubernetes Project's review and pull request processes.
+
+このリポジトリは、(いくつかの)Kubernetsのプロジェクトにおけるレビューやプルリクエストのプロセスに慣れるために、Kubernetesアップストリームトレーニング参加者の安全な場所として使います。
+
+Each subfolder under `japan/` contains assets for kubernetes upstream training holded in event at Japan.
```

Comment:
nit: s/event/events/

Reply:
Thank you to comment. I've fixed this.

Reply:
Thanks for updating.
The Japanese translation on README.md also seems good for me.

/lgtm